### PR TITLE
typos in the source

### PR DESCRIPTION
**Pull request by trebmuh**
While doing the FR translation, I spotted those which looks to be typos to me. Please double-check carefully.

### CODE REVIEW

File: lib-src/mod-nyq-bench/NyqBench.cpp [hunk 1]
```diff
@@ -1721,7 +1721,7 @@ void NyqBench::RecreateToolbar(bool large)
    tb->AddSeparator();
    tb->AddTool(wxID_FIND, _("Find"), mPics[12], _("Find text"));
    tb->AddSeparator();
-   tb->AddTool(ID_MATCH, _("Match"), mPics[13], _("Go to matching paren"));
+   tb->AddTool(ID_MATCH, _("Match"), mPics[13], _("Go to matching parent"));
```
Comment (Pokechu22) [inline lib-src/mod-nyq-bench/NyqBench.cpp]: This is correct as-is; paren is short for parenthesis.

File: src/effects/Effect.cpp [hunk 2]
```diff
@@ -3061,7 +3061,7 @@ bool EffectUIHost::Initialize()
                   bb->SetBitmapDisabled(CreateBitmap(effect_ffwd_disabled_xpm, true, true));
                   mFFwdBtn = bb;
 #if defined(__WXMAC__)
-                  mFFwdBtn->SetName(_("Skip &Foreward"));
+                  mFFwdBtn->SetName(_("Skip &Forward"));
 #else
                   mFFwdBtn->SetLabel(_("Skip &Foreward"));
```
Comment (Pokechu22) [inline src/effects/Effect.cpp]: This does look like a typo.  However, the same mistake happened twice (once for `__WXMAC__`, once outside); you've changed the first but not the second.

Comment (trebmuh) [reply src/effects/Effect.cpp]: Thanks. Done.

File: lib-src/mod-nyq-bench/NyqBench.cpp (hunk 1 above)
Comment (trebmuh) [reply lib-src/mod-nyq-bench/NyqBench.cpp]: OK. My bad then. I've reverse it. Thanks.

**Comment by trebmuh** [comment]
@Pokechu22 : commits incorporates your comment. Thank you.

**Comment by trebmuh** [comment]
Plus two fixes for the same 2 typos in comments lines.

**Comment by trebmuh** [comment]
Another typo found.

**Comment by trebmuh** [comment]
Consistency. Really a small one. Quality sake.

**Comment by Paul-Licameli** [comment]
Thanks.

See commit 0d89bc3f150384916bb0a02172b124ec1ef1debc.

**Pull request closed without merging**